lib/ir.js

```diff
@@ -78,11 +77,11 @@ Devices = {
       };
 
       // if ( temp.left < 200 ) {
-      //   this.emit( "left", err, timestamp );
```

Comment:
Actually, I think this whole file will be refactored soon, so I'm going to leave this in for now (out of sheer laziness)